clients/src/main/java/org/apache/kafka/clients/producer/Callback.java

```diff
@@ -52,6 +54,7 @@ public interface Callback {
      *                      <li>{@link org.apache.kafka.common.errors.OffsetOutOfRangeException OffsetOutOfRangeException}
      *                      <li>{@link org.apache.kafka.common.errors.TimeoutException TimeoutException}
      *                      <li>{@link org.apache.kafka.common.errors.UnknownTopicOrPartitionException UnknownTopicOrPartitionException}
+     *                      <li>{@link BufferExhaustedException BufferExhaustedException}
```

Comment:
For consistency, you should use the full `org.apache.kafka.clients.producer.BufferExhaustedException` class name.